fmi_weather_client/parsers/forecast.py

```diff
@@ -147,7 +147,7 @@ def to_value(vals: Dict[str, float], variable_name: str, unit: str) -> Value:
         )
 
 
-def _feels_like(vals: Dict[str, float]) -> float:
+def _feels_like(vals: Dict[str, float]) -> float | None:
```

Comment:
This is a nice change, but unfortunately Union types were added in Python 3.10, so they can't be used as long as 3.9 is supported. Maybe later this year ☺️ More info [here](https://docs.python.org/3/library/stdtypes.html#union-type).

Reply:
As long as it keeps the code backward compatible with Python 3.9, I'm fine with it ☺️